src/mappers/mapLiteral.ts

```diff
@@ -39,7 +39,7 @@ export default function mapLiteral(context: ParseContext, node: Literal): Node {
     return new JavaScript(line, column, start, end, raw, virtual, node.value);
   }
 
-  if (startToken.type === SourceType.NUMBER) {
+  if (startToken.type === SourceType.NUMBER || startToken.type === SourceType.DOT) {
```

Comment:
I guess this leads to the question: should these be two tokens? Or just a single `NUMBER` token.